native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/AbstractStatementParameterProcessor.java

```diff
@@ -486,4 +491,8 @@ private int getSQLType(BObject typedValue) throws DataError, SQLException {
         return sqlTypeValue;
     }
 
+    private int setBMapParams(Connection connection, PreparedStatement preparedStatement, int index,
+                                               Object value, boolean returnType) throws DataError, SQLException {
+        return setCustomBMapParams(connection, preparedStatement, index, value, returnType);
```

Comment:
```suggestion
        return setCustomBOpenRexord(connection, preparedStatement, index, value, returnType);
```

Reply:
corrected in https://github.com/ballerina-platform/module-ballerina-sql/pull/358/commits/cd74b3fe593a9cc709b97184cdbe4d6f82d9aea0